PhysLean/ClassicalMechanics/FreeParticle/LandauDerivation.lean

```diff
@@ -0,0 +1,419 @@
+/-
+Copyright (c) 2025 Joseph Tooby-Smith. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joseph Tooby-Smith
```

Comment:
Maybe this should include just "and contributors" instead of my name so that the header is uniform across files?